ISICkonto/AppDelegate.swift

```diff
@@ -7,40 +7,74 @@
 //
 
 import UIKit
+import SVProgressHUD
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
-
-
+    var appFlowController: AppFlowController?
+    
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        setUpGlobalAppearance()
+        
+        window = UIWindow(frame: UIScreen.main.bounds)
+        
+        if let window = window {
+            window.rootViewController = UINavigationController()
+            appFlowController = AppFlowController(with: window.rootViewController as! UINavigationController)
+            appFlowController!.launch()
```

Comment:
vykřičníky jsou většinou špatně, tady by to klidně šlo zavolat na `appFlowController?.launch()` a výsledek by byl stejný 😊

Reply:
editted